docker/pravega/scripts/ltsrecovery.sh

```diff
@@ -0,0 +1,114 @@
+IFS='
```

Comment:
The name of the file may be misleading, as it would look like we are recovering LTS itself. It would be better something like `recovery_pravega_from_lts.sh` or something of the sort.

Reply:
renamed it to `pravega_recovery_setup.sh` as the script just performs the setup of the recovery pod and doesnt actually execute the recovery procedure.